daq/topology.py

```diff
@@ -1,5 +1,4 @@
 """Faucet-specific topology module"""
-
```

Comment:
prevailing code convention is to have the blank line after the initial comment... was something complaining about this?